testsuite/MDAnalysisTests/coordinates/test_trc.py

```diff
@@ -0,0 +1,189 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose
+from numpy.testing import assert_equal
+
+import MDAnalysis as mda
+from MDAnalysis.coordinates.TRC import TRCReader
+from MDAnalysisTests.datafiles import TRC_PDB_VAC, TRC_TRAJ1_VAC, TRC_TRAJ2_VAC
+from MDAnalysisTests.datafiles import TRC_CLUSTER_VAC
+from MDAnalysisTests.datafiles import TRC_PDB_SOLV, TRC_TRAJ_SOLV
+
+
+class TestTRCReaderVacuumBox:
```

Comment:
You are testing a chainreader here, could you test just a normal single file for most of the files, and then go do chainreader tests seperately?

Reply:
I don't get the task here. 
`TestTRCReaderVacuumBox` tests the behavior with the chainreader, `TestTRCReaderSolvatedBox` and `TestTRCReaderClusterTrajectory` with a single trajectory file.

Should all tests which are being done with the chainreader, also be done with a single file trajectory?

Please note, that with the GROMOS file structure, the chainreader is used almost exclusively and should be considered as the standard case.

Reply:
I would think that we don't need the same test with chainreader and single trajectory so if we have separate tests for chainreader and single traj then we should be fine. @hmacdope please do correct me!

Reply:
I think you have covered the testing well enough, I think I confused myself, ignore me.